parameters_meta_schema.json

```diff
@@ -69,12 +78,44 @@
                     ],
                     "properties": {
                       "type": {
-                        "type": "string",
-                        "enum": ["string", "boolean", "integer", "number"]
+                        "anyOf": [
+                          { 
+                            "$ref": "#/$defs/typeAnnotation"
+                          },
+                          {
+                            "type": "array",
+                            "items": {
+                              "$ref": "#/$defs/typeAnnotation"
+                            }
+                          }
+                        ]
                       },
                       "format": {
                         "type": "string",
-                        "enum": ["file-path", "directory-path", "path", "file-path-pattern"]
+                        "enum": [
+                          "file-path",
+                          "directory-path",
+                          "path",
+                          "file-path-pattern",
+                          "date-time",
+                          "date",
+                          "time",
+                          "duration",
+                          "email",
+                          "idn-email",
+                          "hostname",
+                          "id-hostname",
+                          "ipv4",
+                          "ipv6",
+                          "uuid",
+                          "uri",
+                          "uri-reference",
+                          "iri",
+                          "iri-reference",
+                          "uri-template",
+                          "json-pointer",
+                          "regex"
+                        ]
```

Comment:
I don't think that we do want all of these though.

Although JSON schema libraries can validate them, the launch form / schema builder / other tooling wouldn't know what to do with them.

Not sure that there's any point allowing them for validation if they will break other tooling. Any of these things can still be set as a string without `format` being set.

Reply:
> I'm not sure having the other tooling break on an unknown format is that great of an idea.

If they're listed here then other tooling would accept the schema but then not know what to do with the `format` and either ignore it or break. So I'm suggesting slimming it down for the same reason 😅 

My expectation of how this will work is that tools will check the schema against the meta schema and show a warning if it fails. So for example:

```
Warning: Schema seems invalid - value 'ipv6' not allowed for 'format'
```

It would then hopefully still continue to try to use the schema as best it can. Which I think is better than including them all, not knowing in advance, and then breaking.

Reply:
Okay so I think these make the most sense to keep:
```
[
                          "file-path",
                          "directory-path",
                          "path",
                          "file-path-pattern",
                          "date-time",
                          "date",
                          "time",
                          "email",
                          "uri",
                          "regex"
                        ]
```

Do you agree or should I also remove the ones that aren't built-in to the plugin?